src/screens/home/HomeScreen.tsx

```diff
@@ -1,21 +1,98 @@
 import React from 'react';
-import {StyleSheet, Text, View} from "react-native";
-import {StatusBar} from "expo-status-bar";
+import { Platform, SafeAreaView, StatusBar, StyleSheet, View, ScrollView } from 'react-native';
 
-export const HomeScreen = () => {
-    return (
-        <View style={styles.container}>
-            <Text>Open up App.tsx to start working on your app!</Text>
-            <StatusBar style="auto" />
+import { CardItem } from '@/components/CardItem';
+import { COLORS, SPACING } from '@/theme/theme';
+
+const mockItemData = [
+  {
+    id: '1',
+    name: 'Product 1',
+    image: {
+      uri: 'https://img.freepik.com/free-photo/top-view-pepperoni-pizza-with-mushroom-sausages-bell-pepper-olive-corn-black-wooden_141793-2158.jpg',
+    },
+    prices: 5,
+    isNew: true,
+    description: 'Description for Product 1',
+  },
+  {
+    id: '2',
+    name: 'Product 2',
+    image: {
+      uri: 'https://img.freepik.com/free-photo/top-view-pepperoni-pizza-with-mushroom-sausages-bell-pepper-olive-corn-black-wooden_141793-2158.jpg',
+    },
+    prices: 4,
+    isNew: false,
+    description: 'Description for Product 2',
+  },
+  {
+    id: '3',
+    name: 'Product 3',
+    image: {
+      uri: 'https://img.freepik.com/free-photo/top-view-pepperoni-pizza-with-mushroom-sausages-bell-pepper-olive-corn-black-wooden_141793-2158.jpg',
+    },
+    prices: 10,
+    isNew: false,
+    description: 'Description for Product 3',
+  },
+  {
+    id: '4',
+    name: 'Product 4',
+    image: {
+      uri: 'https://img.freepik.com/free-photo/top-view-pepperoni-pizza-with-mushroom-sausages-bell-pepper-olive-corn-black-wooden_141793-2158.jpg',
+    },
+    prices: 10,
+    isNew: true,
+    description: 'Description for Product 4',
+  },
+  {
+    id: '5',
+    name: 'Product 5',
+    image: {
+      uri: 'https://img.freepik.com/free-photo/top-view-pepperoni-pizza-with-mushroom-sausages-bell-pepper-olive-corn-black-wooden_141793-2158.jpg',
+    },
+    prices: 10,
+    isNew: true,
+    description: 'Description for Product 5',
+  },
+];
+
+const HomeScreen: React.FC<any> = () => {
+  return (
+    <SafeAreaView style={styles.safeArea}>
+      <ScrollView>
+        <View style={styles.listItemContainer}>
+          {mockItemData.map((el, index) => (
+            <CardItem
+              style={styles.itemContainer}
+              key={index}
+              id={el.id}
+              name={el.name}
+              image={el.image}
+              prices={el.prices}
+              isNew={el.isNew}
+              description={el.description}
+            />
+          ))}
         </View>
-    );
+      </ScrollView>
+    </SafeAreaView>
+  );
 };
 
 const styles = StyleSheet.create({
-    container: {
-        flex: 1,
-        backgroundColor: '#fff',
-        alignItems: 'center',
-        justifyContent: 'center',
-    },
+  safeArea: {
+    flex: 1,
+    paddingTop: Platform.OS === 'android' ? StatusBar.currentHeight : 0,
```

Comment:
You can try using `react-native-safe-area-context`
https://docs.expo.dev/versions/latest/sdk/safe-area-context/#context
Wrap your app with `<SafeAreaProvider>` and then your screen with `<SafeAreaView>` (import should be from  `react-native-safe-area-context`)
It will also fix styles in iPhone devices 
Then you can remove `paddingTop` style.